backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlTableMetadata.java

```diff
@@ -297,7 +289,7 @@ public void dropColumn(String name) {
       throw new MolgenisException("Drop column " + name + " failed: column does not exist");
     }
     // if changing 'ref' then check if not refBack exists
-    checkNotRefback(name, column);
```

Comment:
I think you need to move this check so that it fires within transaction? 
I expected this to be checked within 'executeRemoveColumn' so that when executeDropTable is executed it also happens.

Reply:
This functionality was already in place. Tried moving it to executeRemoveColumn breaks a lot of stuff